src/components/a-radio/_preview.html

```diff
@@ -1,5 +1,9 @@
 <axa-radio value="a" name="test-name" >hello</axa-radio>
 <axa-radio value="b" name="test-name" checked>hello</axa-radio>
-<axa-radio value="c" name="test-name" disabled>hello</axa-radio>
-<axa-radio value="d" name="test-name" error>hello</axa-radio>
+<axa-radio value="c" name="test-name" disabled>
+  hello hello <a href="#" target="_blank">hello hello hello hello hello hello hello hello</a> hello hello hello hello <a href="#" target="_blank">hello hello hello hello </a> hello hello hello hello.
```

Comment:
please don't commit these extensive test example

Reply:
I think is ok extensive example, but maybe use lorem ipsum